docs/permission/plugin-authors/03-adding-a-resource-permission-check.md

```diff
@@ -62,41 +58,48 @@ To start with, let's edit `plugins/todo-list-backend/src/service/router.ts` in a
 +         },
 +       )
 +     )[0];
-
++
 +     if (decision.result !== AuthorizeResult.ALLOW) {
 +       throw new NotAllowedError('Unauthorized');
 +     }
-+
+
       res.json(update(req.body));
     });
 ```
 
-**Important:** Notice that we are passing an extra `resourceRef` object, containing the `id` of the todo we want to authorize.
+**Important:** Notice that we are passing an extra `resourceRef` field, with the `id` of the todo item as the value.
 
-This enables decisions based on characteristics of the resource, but it's important to note that to enable authorizing multiple resources at once, **the resourceRef is not passed to PermissionPolicy#handle**. Instead, policies must return a _conditional decision_.
+This enables decisions based on characteristics of the resource, but it's important to note that in order to enable authorizing multiple resources at once, **the resourceRef is not passed to PermissionPolicy#handle**. Instead, policies must return a [_conditional decision_](../writing-a-policy.md#conditional-decisions).
 
-Before diving into the extra steps needed for supporting conditional decisions, let's go back to the permission policy's handle function used by your adopters and try to authorize our new permission.
+Before diving into the extra steps needed to support conditional decisions, let's go back to the permission policy's handle function used by your adopters and try to authorize our new permission.
```

Comment:
What was the reasoning to update the policy before defining the `isOwner` rule? Organizing the steps this way requires an explanation of an error below that would've been avoided if we defined the permission rule and set up the integration router before updating the policy

Reply:
I think I'm on the same train here. Will update!

Reply:
Changed in [this commit](https://github.com/backstage/backstage/pull/10782/commits/dbbd50db6589c950f79d518833a972f091c78547)